honeybee_vtk/legend_parameter.py

```diff
@@ -24,39 +26,70 @@ class Orientation(Enum):
     horizontal = 'horizontal'
 
 
-class Colors(Enum):
+class ColorSet(Enum):
```

Comment:
I understand that you are trying to be subtle here but it makes things pretty confusing when you have two classes that are named `Colorset` and you sonly changed the capitalization of one letter. Maybe you should call this enumeration something else. Even calling it `Colorsets` would be better.

Reply:
Complied